internal/serve/serve.go

```diff
@@ -56,6 +58,11 @@ type Configs struct {
 	HorizonClientURL                   string
 	DistributionAccountSignatureClient signing.SignatureClient
 	ChannelAccountSignatureClient      signing.SignatureClient
+
+	// Error Tracker
+	AppTracker         sentry.SentryTracker
+	SentryDsn          string
+	StellarEnvironment string
```

Comment:
Here is the same comment I've made for the Ingestion configs, we can use local variables for the sentry DSN.

Reply:
Done